### PR TITLE
[Elasticsearch-connector] Autocomplete suggestions

### PR DESCRIPTION
**Pull request by joemcelroy**
This PR adds autocomplete functionality when the developer uses the elasticsearch-connector. This uses Searchkit autocomplete suggestors (https://searchkit.co/docs/core/reference/searchkit-sdk/query-autocomplete) which behind the scenes, uses the bool prefix query and completion suggestor in Elasticsearch query DSL. 

Ive updated the documentation for both the elasticsearch connector page and the elasticsearch tutorial page.

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _docs-elastic-co_ successfully!

* https://docs-elastic-alhm4iwo7-elastic-dev.vercel.app

* Built with commit 07732313fa7bd6368d01abe254f2c84ae19b5cf1

> Issues? Visit #next-docs in Slack

File: docs/tutorials-elasticsearch.mdx [hunk 1]
```diff
@@ -204,6 +216,16 @@ PUT /my-example-movies/_mapping
 
 Elasticsearch will acknowledge the request in the response.
 
+We also want to provide autocomplete functionality, so we need to setup fields for autocomplete.
```
Comment (yakhinvadim) [inline docs/tutorials-elasticsearch.mdx:219]: Awesome job on updating the main tutorial!

File: examples/sandbox/src/pages/root.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ export default function Root() {
         <li>
           <Link to="/elasticsearch">Elasticsearch</Link>
         </li>
+        <li>
+          <Link to="/elasticsearch-ecommerce">Elasticsearch E-Commerce</Link>
+        </li>
```
[the remark this answers is not in the document]
Comment (joemcelroy) [reply examples/sandbox/src/pages/root.js:14]: Thanks for flagging. Had my own concerns too. decided to remove the page and make the us-parks demo have a working autocomplete.

**Comment by yakhinvadim** [comment]
I cannot trigger query suggestions locally, do we only have result suggestions in the elasticsearch example?
image

**Comment by joemcelroy** [comment]
> I cannot trigger query suggestions locally, do we only have result suggestions in the elasticsearch example? 

Try a US state 
image

**Review by yakhinvadim — APPROVED** [review]

**Pull request merged**